plat/kvm/efi.c

```diff
@@ -340,7 +350,22 @@ static void uk_efi_setup_bootinfo_mrds(struct ukplat_bootinfo *bi)
 		uk_efi_crash("Failed to free rt_mrds\n");
 
 	/* Get memory map through GetMemoryMap */
-	uk_efi_get_mmap(&map_start, &map_sz, &desc_sz);
+	uk_efi_get_mmap(&uk_efi_map_key, &map_start, &map_sz, &desc_sz);
+
+	/* We now exit Boot Services since we no longer need it */
+	/* In case of exit failure, we obtain the memory map again, since the
+	 * memory map may have been changed
+	 */
+	status = uk_efi_bs->exit_boot_services(uk_efi_sh, uk_efi_map_key);
+	if (unlikely(status != UK_EFI_SUCCESS)) {
+		uk_efi_get_mmap(&uk_efi_map_key, &map_start, &map_sz, &desc_sz);
+		status =
+		    uk_efi_bs->exit_boot_services(uk_efi_sh, uk_efi_map_key);
```

Comment:
Let's keep the function call on the same line as `status = ` and, instead, place the second argument on the second line (if it is still needed after the rename).
```C
status = uk_efi_bs->exit_boot_services(uk_efi_sh,
                                       map_key);
```